gapic/src/main/com/google/gapid/views/PerformanceView.java

```diff
@@ -101,6 +101,10 @@ public void onCommandsLoaded() {
 
   @Override
   public void onProfileLoaded(Loadable.Message error) {
+    if (error != null || models.profile.getData() == null) {
+      loading.showMessage(Error, Messages.PROFILE_LOAD_FAILURE);
```

Comment:
Why not just `loading.showMessage(error)`?

Reply:
When I was writing I was thinking 
1. respect the convention of how we handled `onCaptureLoaded` failure above (I thought it was written that way because we don't want to show that detail in UI?), 
2. still show something when this if clause is triggered by the second predicate `models.profile.getData() == null` (so far it is sure that if data is null after loading, there should definitely be an error instance accompanied by, but I'm worried about possible corner cases in the future)

I can append the actual error message if you think it's necessary!

Reply:
I think the capture error is handled that way, because the error is displayed somewhere else, because the capture model isn't really this view's primary model. However, the profile is this view's primary model, so we should show the error from the model, rather than a generic message.
Assuming that the model is loaded properly (i.e. not null) if the event is fired and `error` is null, is fine. We do that all over the place.

Reply:
Great! Updated the PR to do simply the error check, and reflect the original error message on UI, thanks for explaining!